src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -279,25 +280,27 @@ class CLEVAScenario(Scenario):
     def __init__(
         self,
         version: str,
-        task: str,
         subtask: str,
         prompt_template: Dict[str, Any],
     ):
         """
         Initializes CLEVA scenario.
         Args:
             version: String identifier for version in a format of 'v[1-9]*([0-9])'.
-            task: String identifier for task.
             subtask: String identifier for subtask.
             prompt_template: Prompt template in CLEVA format.
         """
         super().__init__()
-        self.task = task
         self.subtask = subtask
         self.version = version
         self.converter = Converter()
         self.prompt_template = prompt_template
 
+    @property
+    @abstractmethod
+    def task(self) -> str:
+        raise NotImplementedError("CLEVAScenario is intended to be an abstract base class and cannot be used directly.")
```

Comment:
When defining an abstract method with `@abstractmethod`, I think a simple `pass` should suffice because the `@abstractmethod` decorator itself ensures that the method cannot be called unless it's overridden by a concrete subclass.

Reply:
Fixed.